### ISSUE TITLE
Problem with opengraph description

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#9203]
### Describe the problem and steps to reproduce it:

Extract the OG metadata present in this page: https://addons.mozilla.org/en-US/firefox/user/15360506/

### What happened?

The description is broken:

```
<meta data-react-helmet="true" property="og:description" content="The profile of , Firefox extension author. Find other extensions by , including average ratings, tenure, and the option to report issues.">
```

### What did you expect to happen?

Not that. The author's name should be present in this description.

### Anything else we should know?

(I'll add more info on Monday)

**Comment by ioanarusiczki** [comment]
@willdurand I see on both dev and stage the display name as being available in og:description metadata.

![the name](https://user-images.githubusercontent.com/33448286/76756377-3a56b780-678e-11ea-99c1-655c453ca3c6.png)

Is there anything else I should check on?

**Comment by willdurand** [comment]
> Is there anything else I should check on?

It's weird that stage is not affected by this issue. Could you try with an old user profile maybe?

Other than that, I think double-checking the user profile linked in the description (from -prod) will be good if we cannot "replicate" in stage.

**Comment by ioanarusiczki** [comment]
@willdurand It's a strange situation the one from prod. My old prod account shows the name in og:description. However I tested again with an old and a new account too (the og:description metadata is available in this case only after submitting an add-on). In all cases the name was displayed. 
I can confirm that your example won't show the name. I hope the fix will help.

Verified on dev with FF74(Win10).
